src/igvfd/tests/test_audit_measurement_set.py

```diff
@@ -196,6 +197,19 @@ def test_audit_readout(
         for error in res.json['audit'].get('NOT_COMPLIANT', [])
     )
 
+    # Assay term should not be equivalent to
```

Comment:
I think the comment is not finished? should not it include "equivalent to readout term"?

Reply:
My bad, fixed.